### DISCUSSION TITLE
mixcr analyze of bcr outputs files that produce unknown file format error

### DISCUSSION THREAD

**Discussion by meg-gits-it** [milaboratory/mixcr#1233] [Q&A] [answered]
![image](https://github.com/milaboratory/mixcr/assets/106427040/6bf2ac1d-1edf-4bc2-a3f2-6d1267bfb5d1)

In attached image, files have an extension ?.clones.tsv

Only two samples- 10Lane1 and 18Lane1 are in the correct file format, the rest of the files cannot be detected by R or Linux and generate No such file or directory. Auto-tab-completion in R generates the file names as ending with ^M.clones.tsv. However, even those file name paths do not work when run using a script.

**Comment by PoslavskySV** [comment, marked as answer]
Hi, 

You need to use *.clones.tsv files: those are the tab delimited output files with clonotypes from MiXCR. Other files are:

 - .clns/.clna: binary file with clones (you can [export](https://mixcr.com/mixcr/reference/mixcr-export/) them in different ways)
 - .vdjca: binary files with raw alignments (you can [export](https://mixcr.com/mixcr/reference/mixcr-export/) them in different ways)
 - *report*: MiXCR textual reports
 
Also, why do you have separate files for different lanes? If those are sequencing lanes from the same sample, you need to analyzem all together (see [here](https://mixcr.com/mixcr/reference/ref-input-file-name-expansion/#one-sample-multiple-lanes)).
